packages/web/instrumentation.ts

```diff
@@ -0,0 +1,14 @@
+import { registerOTel } from "@vercel/otel";
+
+import { getOpentelemetryServiceName } from "~/utils/service-name";
+
+export function register() {
+  registerOTel({
+    serviceName: getOpentelemetryServiceName(),
+    instrumentationConfig: {
+      fetch: {
+        propagateContextUrls: ["*"],
+      },
+    },
```

Comment:
@p0mvn Otel will automatically propagate the context on fetch requests. However, we have to make sure the service is compatible with the header